marinerescue-frontend/src/pages/HomePage.js

```diff
@@ -1,14 +1,137 @@
 import React from 'react';
 
+import './HomePage.scss';
+
+
 import { Link } from "react-router-dom";
 
 class HomePage extends React.Component {
     render() {
         return (
-            <div>
-                <h1>Welcome to Marine Rescue!</h1>
-                <Link to="/classify">Play a Classification Game</Link>
-            </div>
+            <main>
+                <div class="navbar">
+                    <Link to= "/">
+                    <div class="navbar-logo">
+                        <img src="images/logo.png"></img>
+                    </div>
+                    </Link>
+                    <Link to="/leaderboard">Leaderboard</Link>
+                    <Link to="/about">About</Link>
+                    <Link to="/parents">Parents</Link>
+                    <a href="#playnow"><button id="nav-bar-button">Play Now</button></a>
+                    <a href="profile">Profile</a>
+                    {/* <div class="navbar-logo">
+                        <img src="images/profile-logo.png"></img>
+                    </div> */}
+                </div>
```

Comment:
Can you please move this to be its own component <Navbar> so we can re-use it everyone on our website? Thanks!